bundles/org.eclipse.osgi/container/src/org/eclipse/osgi/internal/framework/processor.aliases

```diff
@@ -33,3 +33,4 @@ s390x                                  # IBM System 390 (64-bit)
 v850e                                   # NEC V850E
 x86-64 amd64 em64t x86_64               # 64 bit x86 architecture
 loongarch64 LoongArch64                 # 64 bit loongarch architecture
+riscv64					# 64 bit riscv architecture
```

Comment:
Please use spaces instead of tabs like in the other lines for consistency and replace the only tab in line 21 with a space too.

Reply:
ack.